App/Containers/App.js

```diff
@@ -1,6 +1,7 @@
 import '../Config'
-import React, { Component } from 'react'
+import PushConfig from '../Config/PushConfig'
 import DebugConfig from '../Config/DebugConfig'
+import React, { Component } from 'react'
```

Comment:
We added the PushConfig line, and I grouped all the config lines together. Is that what you were asking?